WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -410,10 +410,12 @@ public SendTabViewModel(WalletViewModel walletViewModel, bool isTransactionBuild
 				catch (HttpRequestException ex)
 				{
 					NotificationHelpers.Error(ex.ToUserFriendlyString());
+					Logger.LogError(ex);
 				}
 				catch (Exception ex)
 				{
 					NotificationHelpers.Error(ex.ToTypeMessageString());
```

Comment:
The UI should display always the `ToUserFriendlyString` because when there is no transactions it fallback to `ToTypeMessageString` anyway.